bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/ImageImpl.java

```diff
@@ -164,6 +166,10 @@ protected void initModel() {
         fileReference = properties.get(DownloadResource.PN_REFERENCE, String.class);
         alt = properties.get(ImageResource.PN_ALT, String.class);
         title = properties.get(JcrConstants.JCR_TITLE, String.class);
+        featuredImageResourcePath = properties.get(PageListItemImpl.PN_FEATURED_IMAGE_RESOURCE_PATH, String.class);
+        if (StringUtils.isNotEmpty(featuredImageResourcePath)) {
+            pageListItem = true;
```

Comment:
I find it weird that the ImageImpl depends on the List component: the image impl should not know about who is calling it, or in which composite component it is being used.

Reply:
right, in the latest commit https://github.com/adobe/aem-core-wcm-components/pull/2367/commits/cf499ba81168a819b1d2f0a963ae644b529c4967 I made this concept of an external image resource more general which might be used also by other components in the future.